src/popup/router/pages/Tip.vue

```diff
@@ -58,13 +58,13 @@
           @click="toConfirm"
           :disabled="
             !note ||
-              amountError ||
-              noteError ||
-              !minCallFee ||
-              !validUrl ||
-              !url ||
-              urlStatus === 'blacklisted' ||
-              note.length > 280
+            amountError ||
+            noteError ||
+            !minCallFee ||
+            !validUrl ||
+            !url ||
+            urlStatus === 'blacklisted' ||
+            note.length > 280
```

Comment:
Not specific for this PR, but we should document somewhere this behavior and the requirements for tip note to be less than 280 characters length.